site/src/pages/release-notes/1.18.0.mdx

```diff
@@ -0,0 +1,125 @@
+---
+date: 2022-08-05
+---
+
+## 🌟 New features
+
+- <type://DocService> can now render Markdown and [Mermaid](https://mermaid-js.github.io/mermaid/)
+  formatted documentation for annotated services #4299 #4304
+
+```java Markdown
+@Description(value = "### Markdown format", markup = Markup.MARKDOWN)
+@Get("/markdown")
+public HttpResponse markdown() {
+  return HttpResponse.of(200);
+}
+```
+
+```java
+@Description(value = "graph TD;\n" +
+                     " A-->B;\n" +
+                     " A-->C;\n" +
+                     " B-->D;\n" +
+                     " C-->D;",
+             markup = Markup.MERMAID)
+@Get("/mermaid")
+public HttpResponse mermaid() {
+  return HttpResponse.of(200);
+}
+```
+
+- It is now possible to bind decorators to gRPC services using <type://GrpcServiceBuilder>. #4311 #4316
+
+```java
+GrpcService.builder()
+  .addService(myGrpcService, List.of(decorator)) // 👈👈👈
+  .build();
+```
+
+- gRPC users can now use an <type://UnframedGrpcErrorHandler> which provides a
+  [richer error model](https://cloud.google.com/apis/design/errors#error_model) for unframed requests.
+
+```java
+GrpcService
+  .builder()
+  .addService(myGrpcServiceImpl)
+  .unframedGrpcErrorHandler(
+    UnframedGrpcErrorHandler.ofRichJson()) // 👈👈👈
+  .build();
+```
+
+```json
+{
+  "code": 2,
+  "message": "A custom message!"
+}
+```
+
+- Users can now extract the innermost element of <type://Unwrappable> type objects easily using <type://Unwrappable#unwrapAll()> #4338
+```java
+Foo foo = new Foo();
+Bar<Foo> bar = new Bar<>(foo);
+Qux<Bar<Foo>> qux = new Qux<>(bar);
+assert qux.unwrapAll() == foo; // 👈👈👈
+```
+
+## 📈 Improvements
+
+- <type://RequestContext> now extends <type://Unwrappable>. #4308
+
+## 🛠️ Bug fixes
+
+- You no longer see an intermittent <type://IllegalStateException> for gRPC services
+  when <type://GrpcServiceBuilder#useBlockingTaskExecutor(boolean)> is enabled. #4367 #4374
+- Requests no longer hang when gRPC-JSON transcoding is enabled and gRPC decorators return an unframed response. #4337 #4348
+- Files from OSGI bundle class loader are now correctly served #4371 #4373
```

Comment:
How about mentioning `HttpFile` because the exception is raised when a user uses it?
```suggestion
- You can now use <type://HttpFile> to serve files from the OSGI bundle. #4371 #4373
```